server/controllers/borrow.js

```diff
@@ -4,7 +4,7 @@ import silver from '../helper/silver';
 
 const bookHistoryController = {
```

Comment:
endeavor to use pascal case when exporting as function library like this one. look [http://airbnb.io/javascript/#naming--PascalCase-singleton](here) for further information.